packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -195,36 +195,31 @@ Use `shorebird flutter versions list` to list available versions.
         final appId = shorebirdYaml.getAppId(flavor: flavor);
         final app = await codePushClientWrapper.getApp(appId: appId);
 
-        final bundleDirPath = p.join(
-          projectRoot.path,
-          'build',
-          'app',
-          'outputs',
-          'bundle',
-        );
+        String? bundlePath;
+        String? apkPath;
```

Comment:
Do these need to be nullable?

Dart supports fancy:
```
final Foo foo;
try {
   foo = thing();
} catch () {
   logger.info('failed');
   return;
}
```
It will do flow analysis to know that the finals are set during the try or not used after.

Reply:
Ah, nevermind. I see they can get explicitly set to null.

Reply:
We can still have the top-level variables be non-nullable and only set the in the try block when we have successfully determined a bundlePath and apkPath.